README.md

```diff
@@ -215,7 +220,11 @@ SELECT * FROM table WHERE col_double = cast(? AS DOUBLE) OR col_timestamp = CAST
 ### Response rows
 
 When reading response rows, the driver supports most Trino data types, except:
-* time and timestamps with precision - all time types are returned as `time.Time`
+* time and timestamps with precision - all time types are returned as `time.Time`.
+  All precisions up to nanoseconds (`TIMESTAMP(9)` or `TIME(9)`) are supported (since
```

Comment:
why exactly are we constraining ourselve to nanosecond precision? Would it be possible to support up to `TIME*(12)`?

Reply:
Because Go's `time.Time` only supports nanoseconds. To support picoseconds, we'd have to define a custom data type. I'm not sure how much work it would be to make it compatible with `time.Time`.

Reply:
Recent work in the Python client in https://github.com/trinodb/trino-python-client/pull/300 is also limited but to microseconds.